app/controllers/api/v2/users_controller.rb

```diff
@@ -23,6 +23,7 @@ class UsersController < V2::BaseController
       param :auth_source_ldap_id, String, :desc => N_("ID of LDAP authentication source")
       param :usergroup_id, String, :desc => N_("ID of user group")
       param :role_id, String, :desc => N_("ID of role")
+      param :cached_usergroups, Hash, :desc => N_("List all user groups included inherited user groups")
```

Comment:
Actually I don't remember, but I thing it is from last commit and I forgot delete it